drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
+                            // so remove the ability to move focus to it using the keyboard
         }
         return (
             h('div.option',
                 {
                     key: item.value,
                     className: item.class_name,
-                    attributes: {'data-value': item.value, 'data-drag-disabled': item.drag_disabled},
+                    attributes: {
+                        'tabindex': tabindex,
+                        'draggable': true,
+                        'aria-grabbed': false,
```

Comment:
This is good, but you also need to extend the 'startDragging' and 'endDragging' events to set this to `aria-grabbed=true` when the block is picked up and actively being dragged.